.cirrus.yml

```
@@ -0,0 +1,68 @@
+linux_aarch64_test_task:
+  compute_engine_instance:
+    image_project: cirrus-images
+    image: family/docker-builder-arm64
+    architecture: arm64
+    platform: linux
+    cpu: 4
+    memory: 16G
```

Also, is this the right setup? It looks like a full-fledged VM, but their [Linux container docs](https://cirrus-ci.org/guide/linux/) say you can just use `arm_container`. We can probably use a `python` image:
```suggestion
  arm_container:
    image: "python:slim"
```

---

I think I tried a few versions and this one is (probably) copied from scipy/numpy.

---

This is indeed copied from scipy. I also note that they are also putting the installation in the `test_script` part...